src/courses/forms.py

```diff
@@ -117,16 +117,31 @@ class Meta:
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
+        self.fields['name'].help_text = 'If empty, will instead use First day as it\'s name'
+        self.fields['name'].required = False
+
         self.helper = FormHelper(self)
         self.helper.include_media = False
         self.helper.disable_csrf = True
         self.helper.form_tag = False
 
         self.helper.layout = Layout(
-            Div('first_day', css_class='col-xs-6 col-sm-4',),
-            Div('last_day', css_class='col-xs-6 col-sm-4',),
+            Div('name', css_class='col-xs-4',),
+            Div('first_day', css_class='col-xs-4',),
+            Div('last_day', css_class='col-xs-4',),
         )
 
+    def save(self, commit=True):
```

Comment:
Not clear to me why this is required...

Reply:
mixed up blank=True and null=True. so the form was giving me an error everytime I updated/created with an empty name field